### ISSUE TITLE
https://gist.github.com/FinnFrotscher/6deea36f845a52e8b2a11295b014101

### ISSUE THREAD

**Issue by FinnFrotscher** [michelson/dante2#75]
I did multiple iterations on this file. each leading to the same error.
they are all variations to your `https://github.com/michelson/dante2-tests/blob/master/index.js`

# the files
[gist](https://gist.github.com/FinnFrotscher/6deeaa36f845a52e8b2a11295b014101)

# the error
```

Uncaught (in promise) Error: addComponentAsRefTo(...): Only a ReactOwner can have refs. 
You might be adding a ref to a component that was not created inside a component's `render` method, 
or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```
# behaviour
when text is selected, no tooltip appears
# possible cause
I use the `/lib/` version instead of the `/es/` #73

**Comment by zcmgyu** [comment]
Same here

**Comment by michelson** [comment]
hi , you have a custom implementation , I really don't know as as far I can see where the problem could be.  take a look at how https://github.com/michelson/dante2-tests is made

**Comment by zcmgyu** [comment]
Hi, Michelson. I release that dante-tests also doesn't t work at all. 
And I can't see react & react-dom in your package.json. Maybe it's main problem. Your project needn't react & react-dom dependence. 
If we add react in my package, it becomes multiples copies.

```
ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/src/model/index.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 55:13-49
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/popovers/image.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 23:13-62
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/popovers/link.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 27:12-60
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/popovers/addButton.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 31:17-70
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/popovers/toolTip.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 35:15-66
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/blocks/image.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 39:14-61
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/blocks/embed.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 43:13-60
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/blocks/video.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 47:13-60
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js

ERROR in ./index.js
Module not found: Error: Can't resolve 'Dante2/es/components/blocks/placeholder.js' in '/Users/zcmgyu/Desktop/Aptech i13/React/React/RichEditor/dante2-tests'
 @ ./index.js 51:19-72
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js
```

**Comment by michelson** [comment]
it seems you are using a version from git ? if it's so you will need to build the modules yourself, so: `cd ./node_modules/Dante2` then `yarn build` 

I've released a new version 0.4.0 with some fixes on how modules are exported along with dante2-tests. please try it out, if something is wrong please reopen this issue

best!
